src/Worker/ClearCache.php

```diff
@@ -35,21 +34,12 @@ class ClearCache
 	public static function execute()
 	{
 		$a = DI::app();
-		$last = DI::config()->get('system', 'cache_last_cleared');
-
-		if ($last) {
-			$next = $last + (3600); // Once per hour
-			$clear_cache = ($next <= time());
-		} else {
-			$clear_cache = true;
-		}
-
-		if (!$clear_cache) {
-			return;
-		}
 
 		// clear old cache
 		DI::cache()->clear();
+		if (!DI::config()->get('system', 'optimize_tables')) {
```

Comment:
Why did you reverse the original condition?

Reply:
Because of bad copy and paste :grimacing: